test/ctia/http/routes/indicator_test.clj

```diff
@@ -256,3 +198,51 @@
                   (map :parsed-body)
                   (map #(dissoc % :id :created :modified :owner))
                   set)))))))
+
+(def api-key "45c1f5e3f05d0")
+(defn redprintln [& s]
+  (print "\u001b[31m")
+  (apply println s)
+  (print "\u001b[0m"))
```

Comment:
This is only for testing.
But yes `log/error` would definitively feel better here.